{
  "CVE Code": "CVE-2013-4188",
  "Published": "2014-03-11T15:00:00",
  "Updated": "2024-08-06T16:38:02.143Z",
  "Description": "traverser.py in Plone 2.1 through 4.1, 4.2.x through 4.2.5, and 4.3.x through 4.3.1 allows remote attackers with administrator privileges to cause a denial of service (infinite loop and resource consumption) via unspecified vectors related to \"retrieving information for certain resources.\"",
  "ProblemTypes": [
    "n/a"
  ],
  "Affected": [
    {
      "vendor": "n/a",
      "product": "n/a",
      "versions": [
        "n/a"
      ]
    }
  ],
  "Reference": [
    "http://plone.org/products/plone/security/advisories/20130618-announcement",
    "http://plone.org/products/plone-hotfix/releases/20130618",
    "http://seclists.org/oss-sec/2013/q3/261",
    "https://bugzilla.redhat.com/show_bug.cgi?id=978449"
  ]
}